ui/src/app/views/workspace-library.tsx

```diff
@@ -70,19 +71,22 @@ export const WorkspaceLibrary = withUserProfile()
 (class extends React.Component<
   {profileState: { profile: Profile, reload: Function } },
   {currentTab: {title: string, icon: string}, errorText: string,
-    workspaceList: WorkspacePermissions[], workspacesLoading: boolean}> {
+    workspaceList: WorkspacePermissions[], featuredWorkspaces: WorkspacePermissions[],
+    workspacesLoading: boolean}> {
   constructor(props) {
     super(props);
     this.state = {
       currentTab: libraryTabEnums.PUBLISHED_WORKSPACES,
       errorText: '',
+      featuredWorkspaces: [],
       workspaceList: [],
       workspacesLoading: true
     };
   }
 
-  componentDidMount() {
-    this.reloadPublishedWorkspaces();
+  async componentDidMount() {
+    this.loadPublishedWorkspaces();
```

Comment:
I think we should be awaiting on each of these method calls so they run in sequence, right?

Reply:
actually since we are starting on the published workspaces page, and we load featured upon click of the link, i'm thinking that maybe we should just load published on init

Reply:
Oh – good point. I actually thought we were only going to show "featured" workspaces for the Bedford release (I missed this detail)... when we launch next week with a few DRC-approved featured workspaces, will they show up in both the published & featured tab, or only in the featured tab?

We should avoid an "empty screen" scenario where users have to click through to another tab before seeing anything meaningful... if that's how things are currently, then we're all good!

Reply:
both tabs will be shown